tests/prep_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+set -e
+
+CHERIBSD_PORT=10086
```

Comment:
These variables are being set in *(three)?* different places . Can we export them through the `CMakeLists.txt` into `run_comp.sh` ?  Otherwise, any change will have to be mirrored in three places. Either using shell commandline params or  using *CMake* `set(ENV{varname})` or some other such trick ?

Reply:
I've combined the three scripts into one in https://github.com/capablevms/CHERI-ELF-comp/pull/5/commits/796d6a0900e5edff6c627a1bc924cf2cd0248118. Think it's better this way. I also did come up with a way of defining these variables in a single place, if we do want multiple scripts.